api/analytics/analytic_report.py

```diff
@@ -50,9 +50,7 @@ def get_all_rooms_summary(self, query, start_date, end_date):
 
         rooms_no_meetings_df = all_rooms_data_df.loc[
             lambda all_rooms_data_df: all_rooms_data_df['minutes'] == 0].rename(
-                columns={
```

Comment:
Try not to introduce unintended changes.

Reply:
My flake8 was complaining about some of these.